### PR TITLE
Added polish translations for Whisper and Piper add-ons

### PR DESCRIPTION
**Pull request by witold-gren**

### CODE REVIEW

File: piper/translations/pl.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+configuration:
+  voice:
+    name: Głos
+    description: >-
+      Głos Piper do użycia. Głosy są nazywane zgodnie z poniższym opisem
+      scheme: `<language>-<name>-<quality>`.
+  speaker:
+    name: Dzwięk
```
Comment (mokrzesa) [inline piper/translations/pl.yaml:9]:
```suggestion
    name: Dźwięk
```

File: piper/translations/pl.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    description: >-
+      Numer dzwięku, którego należy użyć, jeśli głos obsługuje wiele wariantów dzięków.
```
Comment (mokrzesa) [inline piper/translations/pl.yaml:11]:
```suggestion
      Numer dźwięku, którego należy użyć, jeśli głos obsługuje wiele wariantów dźwięków.
```

File: piper/translations/pl.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      Domyślnie używany będzie pierwszy dzwięk (dzwięk 0).
```
Comment (mokrzesa) [inline piper/translations/pl.yaml:12]:
```suggestion
      Domyślnie używany będzie pierwszy dźwięk (dźwięk 0).
```

**Review by mokrzesa — COMMENTED** [review]
Very good job 👍

File: piper/translations/pl.yaml (hunk 1 above)
Comment (witold-gren) [reply piper/translations/pl.yaml]: Thank you for check my translations 😀

**Review by mokrzesa — APPROVED** [review]
Looks very good 🇵🇱 🇵🇱

**Pull request closed without merging**